packages/repository-rest/src/controllers/crud-controller.ts

```diff
@@ -0,0 +1,85 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  CrudRepository,
+  ValueObject,
+  Entity,
+  DataObject,
+  Options,
+  Filter,
+  Where,
+} from '@loopback/repository';
+
+import {post, get, param} from '@loopback/rest';
+
+/**
+ * Base controller class to expose CrudRepository operations to REST
+ */
+export abstract class CrudController<T extends ValueObject | Entity> {
```

Comment:
I would rename this to RestController; that way we can keep the idea of a CrudController open as a base class for a RestController, a GrpcController, etc.